Tests/Users/Controller_UserController_UserLogin.cs

```diff
@@ -15,7 +15,7 @@ public class Controller_UserController_UserLogin
         [SetUp]
         public void SetUp()
         {
-            User = new User() { Email = "test@gmail.com", Username = "test" };
+            User = new User() { Email = "test@gmail.com", Username = "test", IsActive = true};

```

Comment:
Wil je ook het LoginResult NotActivated testen in die test hieronder?
Dus twee gebruikers hier aanmaken, waarvan 1 niet actief is. En die dan testen

Reply:
ja is goed